### ISSUE TITLE
Plugin doesn't support environment configurations

### ISSUE THREAD

**Issue by VijitCoder** [redhat-developer/intellij-quarkus#1346]
IntelliJ IDEA 2024.1.3 
Runtime version: 17.0.11+1-b1207.24 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.

Quarkus plugin ver. 241.17890.1

Autocompletion (and other features) only works with `application.yaml`. But there are also possible _environment_ configs like `application-dev.yaml` and etc. Plugin doesn't support such configs.

**Comment by fbricon** [comment]
can you check if it works with application.properties?

**Comment by VijitCoder** [comment]
![2024-06-28_19-51](https://github.com/redhat-developer/intellij-quarkus/assets/8523131/c6f4881c-6f5d-4073-b753-bb272150f69e)

_Pay attention to the icons, they also marked which files supported._

**Comment by fbricon** [comment]
241.17890.1 is JetBrains' version.

I managed to reproduce the problem with IntelliJ IDEA Ultimate EAP, which comes with their built-in Quarkus support.
I then installed [Quarkus Tools](https://plugins.jetbrains.com/plugin/13234-quarkus-tools?noRedirect=true) (this plugin) and profile aware files started to work (completion/validation/hover). They have the proper icon.

<img width="1132" alt="Screenshot 2024-06-28 at 15 39 35" src="https://github.com/redhat-developer/intellij-quarkus/assets/148698/a7bdacf0-9c3a-497c-8a15-22b3028f5dbf">
